packages/graphql-config-utilities/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "graphql-config-utilities",
+  "description": "Common utilities for graphql-config",
+  "version": "0.0.0",
```

Comment:
i think this makes sense to start with right? so that lerna can bump to `0.1.0` on first publish?

Reply:
Yeah, but probably bump to 1.0 instead